node/src/api/dto.rs

```diff
@@ -0,0 +1,186 @@
+use serde::{Deserialize, Serialize};
+
+use accounts::Receiver;
+
+use super::serde_utils::BigArray;
+use blockchain::BlockTx;
+use std::str::FromStr;
+use zkvm::TxHeader;
+
+/// Stats about unconfirmed transactions.
+#[derive(Serialize)]
+pub struct MempoolStatusDTO {
```

Comment:
We already have a namespacing tool: module names. Maybe use `dto::MempoolStatus` instead? 

<bikeshed>

Also, I'd use something less bureaucratic, maybe "api::types::MempoolStatus"?

</bikeshed>

Reply:
I think "api::types::MempoolStatus" looks better.